### PR TITLE
[case](mtmv)MTMV hive case

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: regression-test/suites/mtmv_p0/test_hive_olap_mtmv.groovy [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_hive_olap_mtmv", "p0,external,hive,external_docker,external_docker_hive") {
+    String enabled = context.config.otherConfigs.get("enableHiveTest")
+    if (enabled == null || !enabled.equalsIgnoreCase("true")) {
+        logger.info("diable Hive test.")
+        return;
+    }
+        String hms_port = context.config.otherConfigs.get("hms_port")
```
Comment (zfr9527) [inline regression-test/suites/mtmv_p0/test_hive_olap_mtmv.groovy:24]: Indentation style is not uniform

**Comment by zfr9527** [comment]
maybe need to add the case of adding and deleting partitions in olap table

**Review by zfr9527 — APPROVED** [review]
LGTM

**Comment by zddr** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Pull request merged**